packages/browser-wallet/src/popup/pages/Setup/Setup.tsx

```diff
@@ -27,13 +40,33 @@ function Intro() {
                 <p>{t('intro.welcome')}</p>
                 <p>{t('intro.description')}</p>
             </div>
-            <Button
-                className="onboarding-setup__intro-wrapper__continue-button"
-                width="medium"
-                onClick={() => navigate(setupRoutes.passcode)}
-            >
-                {t('continue')}
-            </Button>
+
+            <Form onSubmit={handleSubmit}>
```

Comment:
```suggestion
            <Form onSubmit={() => navigate(setupRoutes.passcode)}>
```
Could be a simplification.